Makefile

```diff
@@ -30,23 +26,28 @@ ifeq ($(UNAME_S),Darwin)
 	MOBY_DOCKER=/Applications/Docker.app/Contents/Resources/bin/docker
 endif
 
-BINARY_FOLDER=$(shell pwd)/bin
-GIT_TAG?=$(shell git describe --tags --match "v[0-9]*")
 TEST_FLAGS?=
 E2E_TEST?=
 ifeq ($(E2E_TEST),)
 else
 	TEST_FLAGS=-run $(E2E_TEST)
 endif
 
-all: compose-plugin
+BUILDX_CMD ?= docker buildx
 
-.PHONY: compose-plugin
-compose-plugin: ## Compile the compose cli-plugin
+all: build
+
+.PHONY: build ## Build the compose cli-plugin
+build:
+	$(eval DESTDIR ?= ./bin/build)
```

Comment:
Sorry.. last question; is there an advantage to doing the `$(eval ..)` here, over having `DESTDIR ?= ./bin/build` at the top of the makefile? If there's no advantage, I think it'd be clearer to move it there at the top; makes it slightly more visible / transparent.

Reply:
`DESTDIR` would be propagated to the bake definition and overrides defaults in https://github.com/docker/compose/blob/23deefd20cc5dd5ebe11161df79ab7a60934ceda/docker-bake.hcl#L27-L34

Have to double-check though.

Reply:
Ah it's not actually, not an env var, so looks good to put it at the top.